### DISCUSSION TITLE
Tailwind classes not recognized under some conditions

### DISCUSSION THREAD

**Discussion by bixgomez** [gregsullivan/_tw#42] [General]
This is strange... I am trying to build a custom page, that displays all posts of a certain type.  Fairly innocuous page based on the `page.php` template.

For some strange reason, none of the Tailwind classes I use on that page are being recognized.

The same thing happens when I replace my main content loop with `the_content();`

**Comment by gregsullivan** [comment]
You might need to restart `npm run watch`—it may not be picking up the new files you added since running the watch command the first time.

**Reply by bixgomez** [reply]
That did not work.  I'd like to send my code, but I don't want to clutter this discussion.  Oh, here goes...
```	<section id="primary">
		<main id="main">

			<?php
			/* Start the Loop */
			while ( have_posts() ) :
				the_post();
				get_template_part( 'template-parts/content/content', 'page' );
			endwhile; // End of the loop.
			?>

			<?php
			$args = array(
					'post_type'=> 'artwork',
					'posts_per_page' => -1,
					'order'    => 'ASC'
			);
			$the_query = new WP_Query( $args );
			if($the_query->have_posts() ) :
				while ( $the_query->have_posts() ) :
					$the_query->the_post();
					get_template_part( 'template-parts/content/teaser--artwork', 'artwork-teaser',
							array(
									'title' => get_the_title(),
									'dimensions' => get_field('dimensions'),
									'description' => get_field('description'),
									'image_id' => get_post_thumbnail_id( $post ),
							)
					);
				endwhile;
				wp_reset_postdata();
			else:
			endif;
			?>

		</main><!-- #main -->
	</section><!-- #primary -->```

**Reply by bixgomez** [reply]
'teaser--artwork.php' simply looks like this: 
<?php
echo '<div class="mb-50">';
$title = isset($args['title']) ? $args['title'] : '';
$dimensions = isset($args['dimensions']) ? $args['dimensions'] : '';
$description = isset($args['description']) ? $args['description'] : '';
$image_id = isset($args['image_id']) ? $args['image_id'] : '';
echo $title;
echo $description;
echo $dimensions;
echo $image_id;
echo '</div>';
?>

**Reply by gregsullivan** [reply]
I don't see any Tailwind classes here. Which ones aren't working?

**Reply by bixgomez** [reply]
`echo '<div class="mb-50">';`

Very odd!  That didn't show up when I pasted it

**Reply by gregsullivan** [reply]
`mb-50` isn't a Tailwind class. See here:

https://tailwindcss.com/docs/margin

(You can use `mb-48` or `mb-52`.)

**Reply by gregsullivan** [reply]
Or you could use `mb-[200px]` or `mb-[12.5rem]`.

**Reply by bixgomez** [reply]
OH JEEZ!  Thanks... Well that's embarrassing!

**Reply by gregsullivan** [reply]
It feels like `mb-50` should be a Tailwind class until you do the math and realize it's not an even rem value, so I've definitely been there!

**Reply by bixgomez** [reply]
Now that I've got autocomplete and suggestions working well, it's easier to not choose a nonexistent class!
